mcr.cpp

```diff
@@ -12,36 +17,43 @@ bool win = false;
     if (game[0][2] == game[1][1] && game[1][1] == game[2][0] && (game[0][2] == 'X' || game[0][2] == 'O')) win = true; 
     return win;
 } 
-int main(){
+
+int main() {
     int i, j;  
     char game[3][3] = {' '}; // Tic-tac-toe
     char player1 = 'X'; 
     char player2 = 'O'; 
     bool turn = true; // false for player 1's turn, true for player 2's turn. Player 1 first.
     cout << "X = Player 1" << endl << "O = Player 2" << endl;
-    for (int n=0; n<9; n++){
+    for (int n = 0; n < 9; n++) {
         turn = !turn; // use the not-operator to change true to false or false to true.
         if (turn == false) 
             cout << "Player 1: "; 
         else
             cout << "Player 2: "; 
         cout << "Which cell to mark? i:[1..3], j:[1..3]: "; 
```

Comment:
Based on the output, I think this should be corrected:        i:[0..2], j:[0..2]